docs/peg_oled_display.md

```diff
@@ -0,0 +1,109 @@
+# Peg Oled Display
+To use this you need to make some changes to your kb.py as well as you main.py I will break it into two sections for you. 
+
+### What you can and cant do
+
+#### Can do
+* Display images
+* Display text
+* Set images or text to react to your layer
+
+
+#### Cant do yet / on the way
+* React to battery percentage
+* React to WPM 
+
+## Required Libs
+you need these frozen into your circuitpython or in a lib folder at the root of your drive.
+* [Adafruit_CircuitPython_DisplayIO_SSD1306](https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_SSD1306)
+* [Adafruit_CircuitPython_Display_Text](https://github.com/adafruit/Adafruit_CircuitPython_Display_Text)
+* [download .mpy versions from here](https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/download/20220415/adafruit-circuitpython-bundle-7.x-mpy-20220415.zip)
+
+
+## kb.py
+Your chosen board may already have these changes done If not you will need to add them.
+
+you need to add SCL and SDA to your keyboard. The pins the oled are connected to on your controller may not be called "SCL"and "SDA" they could be "GP21" and "GP13" for example. The best way to find out what they are is look at the boards pinout.
+```python
+    SCL=board.SCL
+    SDA=board.SDA
+```
+
+
+## Main.py
+These are the changes that need to be made / added to your main.py
+### Config
+no mater how you are going to use the oled you need this part
+```python
+    from kmk.extensions.peg_oled_Display import Oled,OledDisplayMode,OledReactionType,OledData
+    keyboard = KMKKeyboard()
+    # ... Other oled code
+    keyboard.extensions.append(oled_ext) 
+
+```
+### Photos
+So the config for photos is quite simple. Getting the photos maybe not so much. I will explain.
+
+Oled takes 2-3 arguments 
+
+1.  OledData
+    * OledData can take image **or** corner_one,corner_two,corner_three and corner_four
+    * Every item in OledData has 2 fields 
+    * 0: this is the reaction type right now it can be OledReactionType.LAYER or OledReactionType.STATIC
+    * 1: An array of the items you want to show for the reaction. In this example 4 images to switch on the 4 layers
+2. toDisplay this takes a OledDisplayMode TXT or IMG.

```

Comment:
I'll let this one go because it's a technical word

Reply:
let me know if you dont like my solution